elastic/kibana
# [Response Ops][Task Manager] Add bulk update function that directly updates using the `esClient`

Resolves https://github.com/elastic/kibana/issues/187704

## Summary

Creates a new `bulkPartialUpdate` function in the `task_store` class that uses the ES client to perform bulk partial updates instead of the Saved Objects client. Updates the update in the `mget` task claimer to use this new function.

## To verify

Run this branch with the `xpack.task_manager.claim_strategy: 'mget'` and ensure that all tasks are running as expected.
